docs/simplecontroller_config.md

```diff
@@ -0,0 +1,39 @@
+# SimpleControllerConfiguration
+
+## Rate
+
+*float64*
+
+Rate is the idle time coefficient for user actions that will be performed sequentially.
+
+- A rate < 1.0 will run actions at a faster pace.
+- A rate == 1.0 will run actions at the default pace.
+- A rate > 1.0 will run actions at a slower pace.
+
+This value is multiplied with [WaitAfterMs](#WaitAfterMs).
+
+## Actions
+
+*[]ActionDefinition*
+
+Actions are the user action definitions that will be run by the SimpleController.
+
+### ActionDefinition
+
+#### ActionId
+
+*string*
+
+Action name which is mapped to SimpleController's actions. Available actions can be found [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/loadtest/control/simplecontroller/controller.go#L137).
+
+#### WaitAfterMs
+
+*integer*
```

Comment:
To be consistent we might want to include the size and type of the integer like you did with the `float64`.

Reply:
If it's just `int` let's use the exact name of the type to avoid confusion.